test/terraform/main.tf

```diff
@@ -0,0 +1,75 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "materialize_infrastructure" {
+  source = "../../"
```

Comment:
```suggestion
  source = "git::https://github.com/MaterializeInc/terraform-aws-materialize.git"
```

Reply:
This will always pull the latest change from that GH repository, an alternative approach is to publish the module on the Terraform Registry so we could version it easily:

https://registry.terraform.io/namespaces/MaterializeInc